### PR TITLE
restrict access to put bundle and files

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

**Comment by Bento007** [comment]
Still finishing up the unit tests.

File: tests/test_subscriptions.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 import connexion.apis.abstract
 import requests
 
+from tests.auth_tests import TestAuthMixin
```
Comment (xbrianh) [inline tests/test_subscriptions.py]: Is `TestAuthMixin` used anywhere else?

Comment (Bento007) [reply tests/test_subscriptions.py]: currently the tests are broken, I'm working on fixing them.

Comment (Bento007) [reply tests/test_subscriptions.py]: it is now used many places

**Pull request merged**